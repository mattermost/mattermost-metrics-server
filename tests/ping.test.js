import { promisify } from "util";
import lambda from "../src/ping";
const handler = promisify(lambda);

describe(`Ping`, () => {
  test(`Should ping back`, () => {
    const event = {};
    const context = {};

    const result = handler(event, context);
    expect(result).resolves.toMatchSnapshot();
  });
});
