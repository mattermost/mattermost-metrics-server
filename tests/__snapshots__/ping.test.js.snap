// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Ping Should ping back 1`] = `
Object {
  "body": "{\\"message\\":\\"I'm alive!\\"}",
  "statusCode": 200,
}
`;
